vpr/src/base/read_options.cpp

```diff
@@ -2881,6 +2881,20 @@ argparse::ArgumentParser create_arg_parser(const std::string& prog_name, t_optio
         .default_value("map")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    route_timing_grp.add_argument<double>(args.router_initial_acc_cost_chan_congestion_threshold, "--router_initial_acc_cost_chan_congestion_threshold")
```

Comment:
To turn this on by default you should get data on VTR and Koios designs too to make sure it is a good idea in general (at least for the default flows on each).
Can run Koios just at the default 300 channel width, but make sure it isn't hurt.
VTR runs Wmin*1.3; care mostly about the final routing, but you can also check what happens to the binary search.
Also need to check 3D chips (talk to Amin for best test).